examples/python/src/hello-world.py

```diff
@@ -0,0 +1,199 @@
+        'path': '/submit',
+    },
+}
+
+
+@app.route('/manifest.json')
+def manifest() -> dict:
+    return {
+        'app_id': 'hello-world',
+        'display_name': 'Hello world app',
+        'homepage_url': 'https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/python/hello-world',
+        'app_type': 'http',
+        'icon': 'icon.png',
+        'requested_permissions': ['act_as_bot'],
+        'on_install': {
+            'path': '/install',
+            'expand': {
+                'app': 'all',
+            },
+        },
+        'bindings': {
+            'path': '/bindings',
+        },
+        'requested_locations': [
+            '/channel_header',
+            '/command'
+        ],
+        'root_url': os.environ.get('ROOT_URL', default_root_url),
+    }
+
+
+@app.route('/submit', methods=['POST'])
+def on_form_submit():
+    print(request.json)
+    return {'type': 'ok', 'text': f'Hello form got submitted. Form data: {request.json["values"]}'}
+
+
+@app.route('/bindings', methods=['GET', 'POST'])
+def on_bindings() -> dict:
+    print(f'bindings called with {request.data}')
+    return {
+        'type': 'ok',
+        'data': [
+            {
+                # binding for a command
+                'location': '/command',
+                'bindings': [
+                    {
+                        'description': 'test command',
+                        'hint': '[This is testing command]',
+                        # this will be the command displayed to user as /first-command
+                        'label': 'first-command',
+                        'icon': 'icon.png',
+                        'submit': {
+                            'path': '/first_command',
+                            # expand block is optional. This is more of metadata like which channel, team this command
+                            # was called from
+                            'expand': {
+                                'app': 'all',
+                                # if you want to expand team & channel, ensure that bot is added to the team & channel
+                                # else command will fail to expand the context
+                                # 'team': 'all',
+                                # 'channel': 'all',
+                            },
+                        },
+                    },
+                    {   # command with embedded form
+                        'description': 'test command',
+                        'hint': '[This is testing command]',
+                        # this will be the command displayed to user as /second-command
+                        'label': 'second-command',
+                        'icon': 'icon.png',
+                        'bindings': [
+                            {
+                                'location': 'send',
+                                'label': 'send',
+                                'form': form
+                            },
```

Comment:
Can we have a comment explaining that this is how you make a subcommand `/second-command send`? Mainly because there are already plenty of comments explaining things around this area, and this is an important concept to understand imo. Also maybe a comment mentioning that we are including the form here

Reply:
Also I know I said that `form` shouldn't be capitalized, but maybe to make it stand out more, we can make it say `SHARED_FORM`

Reply:
Done